src/core/layertree/qgslayertreemodel.h

```diff
@@ -476,8 +481,10 @@ class CORE_EXPORT QgsLayerTreeModel : public QAbstractItemModel
 
   private slots:
     void legendNodeSizeChanged();
+    void hitTestTaskCompleted();
 
   private:
+    void onHitTestResultsChanged();
```

Comment:
When I see a function called onSomething, I tent to think it's bound to a connected signal, which in this case it isn't. I think I'd prefer a name like handleHitTestResults(), but no big deal if you like this syntax. Raising in case this used to be a slot to begin with and subsequently moved to a private function :)

Reply:
Done